pyvex/block.py

```diff
@@ -144,10 +145,10 @@ def convert_expr(expr):
             elif stmttype is CAS:
                 if stmt.oldLo not in invalid_vals: stmt.oldLo = convert_tmp(stmt.oldLo)
                 if stmt.oldHi not in invalid_vals: stmt.oldHi = convert_tmp(stmt.oldHi)
-            for e in stmt.expressions:
-                convert_expr(e)
+            for i in range(len(stmt.expressions)):
```

Comment:
for i, _ in enumerate(stmt.expressions)?

Reply:
for i, expr in enumerate(stmt.expressions): stmt.expressions[i] = convert_expr(expr)